### PR TITLE
170523 fail without

### PR DESCRIPTION
**Pull request by ppalaga**

### CODE REVIEW

**Comment by ppalaga** [comment]
The failure on Appveyor is kinda expected. I have no idea how to fix it.

File: srcdeps-maven-base/src/main/java/org/srcdeps/mvn/config/ConfigurationProducer.java [hunk 1]
```diff
@@ -53,7 +58,13 @@ public ConfigurationProducer() {
                     Constants.MAVEN_MULTI_MODULE_PROJECT_DIRECTORY_PROPERTY));
         }
         final Path basePath = Paths.get(basePathString).toAbsolutePath();
-        final Path srcdepsYamlPath = basePath.resolve(relativeMvnSrcdepsYaml);
+        final Path deafultSrcdepsYamlPath = basePath.resolve(SRCDEPS_YAML_PATH);
```
Comment (jpkrohling) [inline srcdeps-maven-base/src/main/java/org/srcdeps/mvn/config/ConfigurationProducer.java]: `s/deafultSrcdepsYamlPath/defaultSrcdepsYamlPath/`

File: srcdeps-maven-enforcer/src/main/java/org/srcdeps/mvn/enforcer/SrcdepsEnforcer.java [hunk 2]
```diff
@@ -56,8 +57,105 @@
...
+     * @param properties
+     *            the properties present the current Maven invocation
+     * @return the failure triggering item
+     */
+    static String[] assertFailWith(MavenAssertions failWith, List<String> goals, List<String> profiles,
+            Properties properties) {
+
+        log.debug("Srcdeps Enforcer using failWith {}", failWith);
+        final Set<String> failWithGoals = failWith.getGoals();
+        for (String goal : goals) {
```
Comment (jpkrohling) [inline srcdeps-maven-enforcer/src/main/java/org/srcdeps/mvn/enforcer/SrcdepsEnforcer.java]: What's the source level for this repo? You could replace this code with a beautiful lambda if you are able to use Java 8 ;)

File: srcdeps-maven-enforcer/src/test/java/org/srcdeps/mvn/enforcer/SrcdepsEnforcerTest.java [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+import java.util.Properties;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.srcdeps.core.config.MavenAssertions;
+
+public class SrcdepsEnforcerTest {
+
+    @Test
+    public void assertFailWith() {
```
Comment (jpkrohling) [inline srcdeps-maven-enforcer/src/test/java/org/srcdeps/mvn/enforcer/SrcdepsEnforcerTest.java]: I'm not quite sure what this is doing. Would it be worth adding a javadoc, or a comment?

File: srcdeps-maven-enforcer/src/main/java/org/srcdeps/mvn/enforcer/SrcdepsEnforcer.java (hunk 2 above)
Comment (ppalaga) [inline srcdeps-maven-enforcer/src/main/java/org/srcdeps/mvn/enforcer/SrcdepsEnforcer.java]: > What's the source level for this repo? You could replace this code with a beautiful lambda if you are able to use Java 8 ;)

Java 7, unfortunately.

File: srcdeps-maven-base/src/main/java/org/srcdeps/mvn/config/ConfigurationProducer.java (hunk 1 above)
Comment (ppalaga) [inline srcdeps-maven-base/src/main/java/org/srcdeps/mvn/config/ConfigurationProducer.java]: Thanks, fixing...

**Comment by ppalaga** [comment]
Thanks for the review, @jpkrohling !

**Pull request merged**